src/monitoring/monitoring.cc

```diff
@@ -54,11 +57,15 @@ static const std::initializer_list<double> latencyBuckets =
  12.9, 19.4, 29.1, 43.7, 65.6, 98.5, 147, 221, 332, 498, 748, 1122, 1683, 2525,
  3787, 5681, 8522, 12783};
 
-static const char kClient[] = "client";
-static const char kExport[] = "export";
+// TODO(zhitaoli): is it Ipv4?
+// TODO(zhitaoli): make sure it doesn't crash in corner cases
+static const char kClient[] = "client_ip";
+static const char kServer[] = "server_ip";
+static const char kExport[] = "export";  // export id in Export Block in the conf
+static const char kExportPath[] = "export_path"; // the path in Export Block in the conf
```

Comment:
ExportPath maybe customized and doesn't make sense, which should be replaced with ExportName